packages/deploy/src/core/CoreContracts.ts

```diff
@@ -771,163 +769,189 @@ export default class EvmCoreDeploy extends AbstractCoreDeploy<config.EvmCoreCont
         checklist.assertBeaconProxy(
           replicas[remoteDomain],
-          `${remoteDomain}'s replica`,
+          `Replica for ${remoteDomain} is deployed`,
         );
 
+        const remoteDomainNumber =
+          this.context.mustGetDomain(remoteDomain).domain;
         const assumedDomain = await this.xAppConnectionManager.replicaToDomain(
           replicas[remoteDomain].proxy,
         );
-        checklist.equals(remoteDomainNumber, assumedDomain, 'Remote domain');
+        checklist.equals(
+          remoteDomainNumber,
+          assumedDomain,
+          `Replica for ${remoteDomain} is configured in xAppConnectionManager replicaToDomain`,
+        );
         // domainToReplica
         const assumedReplicaAddress =
           await this.xAppConnectionManager.domainToReplica(remoteDomainNumber);
         checklist.equalIds(
           assumedReplicaAddress,
           replicas[remoteDomain].proxy,
-          'Replica address',
+          `Replica for ${remoteDomain} is configured in xAppConnectionManager domainToReplica`,
         );
-      }
-
-      // watcherPermission
 
-      const remoteConfig = this.context.mustGetDomainConfig(remoteDomain);
-      for (const watcher of remoteConfig.configuration.watchers) {
-        const watcherPermission =
-          await this.xAppConnectionManager.watcherPermission(
-            watcher,
-            remoteDomainNumber,
+        // watcherPermission
+        const remoteConfig = this.context.mustGetDomainConfig(remoteDomain);
+        for (const watcher of remoteConfig.configuration.watchers) {
+          const watcherPermission =
+            await this.xAppConnectionManager.watcherPermission(
+              watcher,
+              remoteDomainNumber,
+            );
+          checklist.equals(
+            true,
+            watcherPermission,
+            `Watcher permission for ${remoteDomain} is set`,
           );
-        checklist.equals(true, watcherPermission, 'Watcher permission');
+        }
       }
     }
 
     //  ========= GovRouter =========
     // GovernanceRouter upgrade setup contracts are defined
     checklist.assertBeaconProxy(
       this.data.governanceRouter,
-      'Governance router',
+      'Governance router is deployed',
     );
     // localDomain
     const govLocalDomain = await this.governanceRouter.localDomain();
-    checklist.equals(this.domainNumber, govLocalDomain, 'Gov domain');
+    checklist.equals(
+      this.domainNumber,
+      govLocalDomain,
+      'GovernanceRouter localDomain is properly configured',
+    );
     // recoveryTimelock
     const recoveryTimelock = await this.governanceRouter.recoveryTimelock();
     checklist.equals(
       ethers.BigNumber.from(
         domainConfig.configuration.governance.recoveryTimelock,
       ).toHexString(),
       recoveryTimelock.toHexString(),
-      'Recovery time lock',
+      'GovernanceRouter Recovery time lock is properly configured',
```

Comment:
```suggestion
      'GovernanceRouter Recovery time lock is properly configured to ${recoveryTimelock} seconds`,
```

Reply:
GovernanceRouter Recovery time lock is ${recoveryTimelock}, expected ${config recovery timelock}